### ISSUE TITLE
Minor Bug in the documentation

### ISSUE THREAD

**Issue by gauravgrover95** [react-native-community/react-native-tab-view#561]
Under section [Scene Map](https://github.com/react-native-community/react-native-tab-view#scenemap), the second _renderScene_ function is incomplete. 

Give code in the documentation:

```
renderScene = ({ route }) => {
  switch (route.key) {
  case 'first':
    return <FirstRoute />;
  case 'second':
    return <SecondRoute />;
  default:
    return null;
  }
```

Code that should be in the documentation:

```
renderScene = ({ route }) => {
  switch (route.key) {
  case 'first':
    return <FirstRoute />;
  case 'second':
    return <SecondRoute />;
  default:
    return null;
  }
};
```

Note: Missing curly bracket and the semicolon

**Comment by gauravgrover95** [comment]
I have generated a pull requested. Please take a look and feel free to notify me if some problem.

**Comment by satya164** [comment]
Hey, I just released a new alpha `2.0.0-alpha.0` of the library. It's rewritten using [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler) and [react-native-reanimated](https://github.com/kmagiera/react-native-reanimated) addresses a many platform specific bugs and performance problems. The documentation is updated as well.

Please try the new version and see if it addresses your issue. If not, please open a new issue following the issue template.
